CMakeLists.txt

```diff
@@ -1,30 +1,43 @@
-project (easi)
-cmake_minimum_required (VERSION 3.13)
+project(easi)
+cmake_minimum_required(VERSION 3.13)
+set(EASI_VERSION_MAJOR 1)
+set(EASI_VERSION_MINOR 0)
+set(EASI_VERSION_PATCH 0)
 
-include (GNUInstallDirs)
-include (CMakePackageConfigHelpers)
 
-enable_language (CXX)
-enable_language (Fortran)
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
 
-option (ASAGI "Enable support for ASAGI" ON)
+enable_language(CXX)
+enable_language(Fortran)
+
+option(ASAGI "Enable support for ASAGI" ON)
+option(IMPALAJIT "Use ImpalaJIT" ON)
+option(IMPALAJIT_LLVM "Use llvm version of ImpalaJIT" OFF)
```

Comment:
Maybe it should be a list of options? I mean it would be nice to make it in the same way how we do it in SeisSol with `host_arch`, `device_arch`, etc.

Reply:
Hm. What do you think about:
One option IMPALA that enables/disables IMPALA and one IMPALA_BACKEND = default, llvm?
(Do you have an idea for a better name for default?)

Reply:
maybe `original`?

Reply:
Sure.

Reply:
Done :)